integration/tests/unowned_cluster/unowned_cluster_test.go

```diff
@@ -66,7 +66,7 @@ var _ = Describe("(Integration) [non-eksctl cluster & nodegroup support]", func(
 	})
 
 	AfterSuite(func() {
-		if !params.SkipDelete {
```

Comment:
wait why is this correct?

Reply:
wait actually yeh. this is fine jake i am not completely rubbish

Reply:
The stack only gets created if `SkipCreate` is true, so we should only delete is if `SkipCreate` is true

Reply:
but what if you wanted it created, but not deleted?

Reply:
then we should switch on both? :see_no_evil:

Reply:
> The stack only gets created if SkipCreate is true
when it's `false` probably

i think skipping delete of things when `SkipDelete` is `true` is the way things should work

Reply:
`!Skip` is already way too many negations to keep together in my head

Reply:
MY BRAIN IS MELTING:

We should only delete the the stack (*different to the cluster*) if:
1. It exists, so if SkipCreate was True it doesn't exist, so we shouldn't delete it
2. SkipDelete is true